src/models/contact.ts

```diff
@@ -0,0 +1,16 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+export interface IContact {};
+export interface IContactModel extends IContact, Document {};
```

Comment:
Why do you need IContactModel extending IContact?
Why not just use IContact?

Reply:
I can. But as I remember from Java Laba it is best practice to use in DAO additional layer to change the database without a problem. I'm not planning to change it now, so I can remove it. But I considered it as best practice.